src/components/modals/ViewCreditReportModal.vue

```diff
@@ -57,6 +57,18 @@
             {{ creditReport.status }}
           </p>
         </li>
+        <li class="list-group-item my-3">
+          <label for="exampleFormControlSelect2">Account Number</label>
+          <p class="display-text">
+            {{ creditReport.accountNo }}
+          </p>
+        </li>
+        <li class="list-group-item my-3">
+          <label for="exampleFormControlSelect2">Bank</label>
+          <p class="display-text">
+            {{ creditReport.bankName }}
```

Comment:
Can we have a Bank list? so we could be consistent with the name.

Reply:
We dont have an updated bank api

Reply:
How should i go about it

Reply:
Just a list on the frontend

Reply:
Done